packages/browser-pool/src/playwright/playwright-plugin.ts

```diff
@@ -174,6 +180,19 @@ export class PlaywrightPlugin extends BrowserPlugin<BrowserType, SafeParameters<
         return browser;
     }
 
+    private _notifyOfFailedLaunch() {
+        let debugMessage = `Failed to launch browser. ${this.launchOptions?.executablePath ? 'Check whether the provided executable path is correct.' : ''} `;
+        if (process.env.APIFY_IS_AT_HOME) {
+            debugMessage += 'Make sure you used the correct Actor template for Playwright: '
```

Comment:
its about using correct docker image, not actor template, i'd not mention the templates here at all

Reply:
I think some new users and those who are not familiar with docker might not make the connection between the wrong docker image and the fact that they can fix the issue by instantiating with a new template. Maybe we could change it from `Make sure` to `Try instantiating with that template ...`? But for now I've removed the mention